python/cudf/cudf/tests/test_rolling.py

```diff
@@ -80,17 +86,23 @@ def test_rolling_dataframe_basic(data, agg, nulls, center):
 
     gdf = cudf.from_pandas(pdf)
 
-    for window_size in range(1, len(data) + 1):
-        for min_periods in range(1, window_size + 1):
-            assert_eq(
-                getattr(
-                    pdf.rolling(window_size, min_periods, center), agg
-                )().fillna(-1),
-                getattr(
-                    gdf.rolling(window_size, min_periods, center), agg
-                )().fillna(-1),
-                check_dtype=False,
-            )
+    try:
+        for window_size in range(1, len(data) + 1):
+            for min_periods in range(1, window_size + 1):
+                assert_eq(
+                    getattr(
+                        pdf.rolling(window_size, min_periods, center), agg
+                    )().fillna(-1),
+                    getattr(
+                        gdf.rolling(window_size, min_periods, center), agg
+                    )().fillna(-1),
+                    check_dtype=False,
+                )
+    except AssertionError as e:
+        if agg == "count" and len(pdf) > 0:
+            pytest.xfail(reason="Differ from pandas behavior here")
+        else:
+            raise e
```

Comment:
Can we run the rolling operations outside the try / except to make sure we only catch the `assert_eq`?